samples/bluetooth/broadcast_audio_assistant/README.rst

```diff
@@ -0,0 +1,51 @@
+.. zephyr:code-sample:: bluetooth_broadcast_audio_assistant
+   :name: Bluetooth: Broadcast Audio Assistant
+   :relevant-api: bt_bap
+
+   Bluetooth: Broadcast Audio Assistant
```

Comment:
This is meant to be a short text description of the sample. It is recommended to word this as if you were completing the sentence "This code sample shows how to ..."). Not particularly fancy but maybe:

```suggestion
   Use LE Broadcast Audio Assistant functionality
```

Reply:
I'll go with that - can't come up with something more fancy atm.  Also, I should add that we plan to extend the functionality during the next month to make it a more usable.